Dockerfile

```diff
@@ -66,6 +65,7 @@ ENV NODE_ENV="production" \
 COPY package.json knexfile.env.js ./
 COPY migrations ./migrations
 COPY seeds ./seeds
+RUN cp ./build/client/assets/service-worker.js ./build/client/
```

Comment:
`service-worker.js` is built using Workbox's `InjectManifest` it ends up in assets because that's the defined output folder for the webpack config. 

```  
output: {
  filename: config.isProduction ? "[name].[chunkhash].js" : "[name].js",
  path: path.resolve(config.isProduction ? config.ASSETS_DIR : __dirname),
  publicPath: "/assets/"
}
```

Because of this config, it ends up `assets/` with all the other files, and maps. It is being moved out of there to `build/client` because that is defined as the `PUBLIC_DIR` (https://github.com/politics-rewired/Spoke/blob/feature/service-worker/Dockerfile#L57) and served by express directly (https://github.com/politics-rewired/Spoke/blob/feature/service-worker/src/server/app.ts#L89). Making any changes to this setup was causing a cascading of failures, like the bundle not being found by the frontend, etc. Trying to access the file through `/assets/service-worker.js` would return the bundle file, and not the service worker. This is the easiest way that work. I expect this to be refactored when we do CDN setup, so that paths can be more appropriate.

Reply:
The browsers handle it automatically! https://web.dev/service-worker-lifecycle/#updates

> Your service worker is considered updated if it's byte-different to the one the browser already has. (We're extending this to include imported scripts/modules too.)